src/utils/utils.js

```diff
@@ -7,14 +7,18 @@ export function getISOfromDatepicker(date) {
 
 //date is of type iso string
 export function getMMDDYYYYfromISO(date) {
-	return date.slice(0, 10)
+	if (date === null) {
+		return "Unknown";
+	} else {
+		return date.slice(0, 10);
+	}
 }
 
 //date is of type iso string
 export function getAge(date) {
 	let parsed = Date.parse(date)
 	let diff = Date.now() - parsed
-	let ageDate = new Date(diff)
```

Comment:
Just noticed this potential error probably caused by accident. 
We would need to change `ageDaete` back to `ageDate`.  @lee3665 
ageDate is used in the return statement. 
fyi @gonzalezlrjesus

Reply:
[Commit](https://github.com/Raise-the-Voices/marturia/commit/a42fb09d028befdf5fba55f00c9b6370547977fd#commitcomment-44091060) @sjaymoon15